user/src/components/UserModal/UserModal.tsx

```diff
@@ -35,8 +56,12 @@ const UserModal: FC<UserModalProps> = ({ isOpen, onClose, user }) => {
         </div>
         <div className="flex flex-col items-center gap-12 rounded-[20px] bg-white p-6 shadow-2xl">
           <div className="flex w-[420px] flex-col items-center justify-center gap-6 py-4">
-            <p className="text-3xl font-bold text-font">{user?.name}</p>
-            <p className="text-base font-medium text-font">{user?.email}</p>
+            <p className="text-3xl font-bold text-font">
+              {user.name || 'ゲスト'}
+            </p>
+            <p className="text-base font-medium text-font">
+              {user.email || 'メールアドレスなし'}
+            </p>
```

Comment:
ゲストとメールアドレスって必要なのかな
なんか理由あったりしたら教えてほしい！

```suggestion
            <p className="text-3xl font-bold text-font">{user.name}</p>
            <p className="text-base font-medium text-font">{user.email}</p>
```

Reply:
なんも取れないことはまずないけど、表示されないのもあれだし一応つけてみた
いらないなら消してもいい